src/pretix/static/pretixcontrol/js/ui/main.js

```diff
@@ -314,6 +314,31 @@ var form_handlers = function (el) {
         dependency.closest('.form-group').find('input[name=' + dependency.attr("name") + ']').on("dp.change", update);
     });
 
+    $("input[name$=vat_id]").each(function () {
+        var dependent = $(this),
+            dependency_country = $(this).closest(".panel-body, form").find('select[name$=country]'),
+            dependency_id_is_business_1 = $(this).closest(".panel-body, form").find('input[id$=id_is_business_1]'),
+            update = function (ev) {
+                if (dependency_id_is_business_1.length && !dependency_id_is_business_1.prop("checked")) {
+                    dependent.closest(".form-group").hide();
+                } else {
+                    dependent.prop("disabled", true);
+                    dependency_country.closest(".form-group").find("label").prepend("<span class='fa fa-cog fa-spin'></span> ");
+                    $.getJSON('/js_helpers/is_country_in_eu/?country=' + dependency_country.val(), function (data) {
```

Comment:
An AJAX call to get a list of 27 two-letter country codes feels like overkill. Maybe we can just pass it to JS as a ``data-`` attribute to the form field or something?

Reply:
You are right, it is now passed as a `data-` attribute.